### DISCUSSION TITLE
Command Line Option to Display Version String

### DISCUSSION THREAD

**Discussion by rabarar** [librespot-org/librespot#614] [Feature Requests]
would be nice to have a --version option to display the current version of the binary

**Comment by rabarar** [comment]
sorry about that!! :)

**Comment by tomodachi** [comment]
```
./librespot --version
```
Will print current version so  I guess this discussion could be marked as "solved? /  implemented?

**Reply by rabarar** [reply]
Works for me! Closing out the way-back issues!!! lol
